docs/private-networks/reference/api/index.md

```diff
@@ -2075,6 +2075,68 @@ curl -X POST --data '{"jsonrpc": "2.0","method": "priv_newFilter","params": ["4r
 }
 ```
 
+</TabItem>
+</Tabs>
+
+### `priv_traceTransaction`
+
+Provides a [transaction trace](../../../public-networks/reference/api#trace_transaction) for a private transaction. 
+
+#### Parameters
+- `transactionHash`: _string_ - the hash of the private transaction to trace.
+- `privacyGroupId`: _string_ - the privacy group ID associated with the transaction
+- `options`: _object_ - request options object with the following fields (all optional and default to `false`). 
```

Comment:
Doesn't require `options` parameter.

Reply:
Thanks @NickSneo. It  is not required but is it optional. Is this like the trace_transaction method? That uses options

Reply:
Not is doesn't use options, neither it is optional.

Reply:
Thanks @NickSneo. I removed it